kqueen/blueprints/api/generic_views.py

```diff
@@ -47,6 +47,8 @@ def check_authorization(self):
 
         # form policy key
         policy = '{}:{}'.format(_class.__name__.lower(), self.action)
+        if policy == 'user:delete':
+            policy = '{}_{}'.format(policy, self.obj.role)
```

Comment:
IMO need to get rid of this 'if' case:
1) wrap it in dict.get(,None)
2) or move it into auth view